lib/node.js

```diff
@@ -1572,7 +1572,8 @@ declare class Process extends events$EventEmitter {
     heapTotal : number;
     heapUsed : number;
   };
-  nextTick(cb : Function) : void;
+  nextTick<A, B, C, D, E, F>(cb: (A, B, C, D, E, F) => mixed, A, B, C, D, E, F) : void;
+  nextTick(cb : Function, ...Array<void>) : void;
```

Comment:
right, arity > 6 is a problem. If I leave the old generic version, then it will match all incorrect cases that fail for the generic version, and we don’t get any errors. 

Is there a way to resolve this?